include/aspect/mesh_deformation/interface.h

```diff
@@ -314,6 +321,20 @@ namespace aspect
          */
         void compute_mesh_displacements ();
 
+        /**
+         * Set up the vector with initial displacements of the mesh
+         * due to the initial topography, as supplied by the initial
+         * topography plugin. We set all entries to the initial topography
+         * based on its surface coordinates, i.e. the initial topography
+         * is not corrected for depth from the surface as it is
+         * for the initial mesh deformation. TODO this is ok for now,
```

Comment:
is this something we need to worry about now?

Reply:
I don't think so, because this will become obsolete once Rene's initial mesh deformation functionality is merged.

Reply:
Does this need to be changed now that #3751 is merged?

Reply:
I don't think it can be changed yet. As long as initial topography can still be prescribed through geometry_model/initial_topography, we need these functions.